metadata_mapper/validator/validator.py

```diff
@@ -239,17 +82,46 @@ def set_validatable_fields(self, fields: list[dict[str, Any]] = [],
 
         Returns list[dict]
         """
```

Comment:
Looks like `merge` is no longer an option here - can't actually add a review comment on line 71 because there's been no change on line 71.

Reply:
Cleaned up the docstring to better reflect the new simplified functionality of this method.